lisa/tools/fdisk.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from typing import cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+
+
+class Fdisk(Tool):
+    @property
+    def command(self) -> str:
+        return "fdisk"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def make_partition(self, disk: str) -> None:
```

Comment:
Add doc string to introduce what this method do, and what the argument is for. Is it the name of new partition?